build fail with electron@7.1.2

hi, 

electron@7.1.2 with node@12.8.1，but node-libcurl not support it.

it build fail

> Could you share some code to reproduce the issue and the steps you took? (+ logs if possible)
> 
> Also, which OS are you running?

export npm_config_runtime=electron
export npm_config_target=params.electron_version
export npm_config_disturl=https://atom.io/download/electron
export npm_config_msvs_version=2015
export npm_config_arch=ia32

yarn install
yarn pregyp rebuild

```
  Easy.cc
  Share.cc
..\src\Easy.cc : warning C4819: 该文件包含不能在当前代码页(936)中表示的字符。请将该文件保存为 Unicode 格式以防止数据丢失 [D:\jenkins_home\workspace\task\third_addon\node-libcurl\build\node_libcurl.vcxproj]
  Multi.cc
..\src\Easy.cc(1618): error C2661: “v8::Object::Set”: 没有重载函数接受 2 个参数 [D:\jenkins_home\workspace\task\third_addon\node-libcurl\build\node_libcurl.vcxproj]
  Curl.cc
  CurlHttpPost.cc
  CurlVersionInfo.cc
  win_delay_load_hook.cc
gyp ERR! build error 
```

https://github.com/JCMais/node-libcurl/blob/e97e32388a3189f80e4a01e373f27296c85a061a/src/Easy.cc#L1618

was using a deprecated V8 Array `Set` function signature, starting in Node 13 the two-argument variant was removed and the only supported variant is the three-argument one

compile error:

```bash
../src/Easy.cc:1618:16: error: no matching member function for call to 'Set'
          arr->Set(arr->Length(), Nan::New<v8::String>(curr->data).ToLocalChecked());
          ~~~~~^~~
/Users/asherwin/Library/Caches/node-gyp/7.1.3/include/node/v8.h:3455:37: note: candidate function not viable: requires 3 arguments, but 2 were provided
  V8_WARN_UNUSED_RESULT Maybe<bool> Set(Local<Context> context,
                                    ^
/Users/asherwin/Library/Caches/node-gyp/7.1.3/include/node/v8.h:3458:37: note: candidate function not viable: requires 3 arguments, but 2 were provided
  V8_WARN_UNUSED_RESULT Maybe<bool> Set(Local<Context> context, uint32_t index,
```

--- a/src/Easy.cc
+++ b/src/Easy.cc
@@ -1615,7 +1615,7 @@ NAN_METHOD(Easy::GetInfo) {
         curr = linkedList;
 
         while (curr) {
-          arr->Set(arr->Length(), Nan::New<v8::String>(curr->data).ToLocalChecked());
+          arr->Set(arr->CreationContext(), arr->Length(), Nan::New<v8::String>(curr->data).ToLocalChecked());
           curr = curr->next;
         }
 

--- a/test/curl/callbacks.spec.ts
+++ b/test/curl/callbacks.spec.ts
@@ -58,13 +58,16 @@ describe('Callbacks', () => {
     })
 
     app.put('/headers', (req, res) => {
-      res.send({
-        headers: req.headers,
-        trailers: req.trailers,
-      })
+
+      req.resume()
 
       req.on('end', () => {
         trailers = req.trailers
+
+        res.send({
+          headers: req.headers,
+          trailers: req.trailers,
+        })
       })
     })
   })

